config/environments/production.rb

```diff
@@ -83,8 +85,12 @@
   config.log_formatter = ::Logger::Formatter.new
 
   # Specify that we're not at the root
-  config.action_controller.relative_url_root = '/app/ppd'
-  config.relative_url_root = '/app/ppd'
+  config.action_controller.relative_url_root = ENV['RELATIVE_URL_ROOT'] || '/app/ppd'
+  config.relative_url_root = ENV['RELATIVE_URL_ROOT'] || '/app/ppd'
+
+  # API location can be specified in the environment
+  # But defaults to the dev service
+  config.api_service_url = ENV['API_SERVICE_URL'] || 'http://localhost:8080'
```

Comment:
Similarly here as in @ijdickinson mentioned above. We should probably change the default here to be a meaningful location where you would usually find the API service. I'm not exactly sure what is the right location though.

Reply:
To me, it depends if we're keeping a `dev` service around. If we are, that should be the default. If not, probably the `production` API, since pre-prod is protected by ssh firewall-ery